### ISSUE TITLE
akka, worksheet and vm options

### ISSUE THREAD

**Issue by schiebel** [scala-ide/scala-worksheet#231]
There seems to be no way to specify jvm executions options. I am trying to add "-Dconfig.file=/Users/myhome/reference.conf" to the JVM options for the worksheet so that I can experiment with Akka within the worksheet. Without some explicit direction to a Akka conf file, I get:

`com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'akka'`

When I try to create an actor system:

-  `import akka.actor.ActorSystem`
-  `val system = ActorSystem("workspace")`

I realize Akka is not a great fit for a worksheet, but this problem should exist even for applications that just use Typesafe's config apparatus. Is there a way to specify VM flags for worksheet execution? If not, could this option be added?

**Comment by dragos** [comment]
No, there isn't such a way yet. I don't think it's particularly hard to do.
